packages/react-scripts/scripts/eject.js

```diff
@@ -189,12 +189,14 @@ inquirer
     console.log(`  Adding ${cyan('Babel')} preset`);
     appPackage.babel = {
       presets: ['react-app'],
-    };
-
-    // Add ESlint config
-    console.log(`  Adding ${cyan('ESLint')} configuration`);
-    appPackage.eslintConfig = {
-      extends: 'react-app',
```

Comment:
As part of our fork we have a standalone `.eslintrc` so that editors can use it for linting integration etc, so removing this section just stops it from being added to `package.json` if you `eject` the app :)